torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1532,7 +1541,11 @@ RegisterOperators reg2({
       Operator(                                                                \
           "prim::DictIndex(Dict(" key_type ", t) self, " key_type              \
           " key) -> t",                                                        \
-          dictIndex)
+          dictIndex),                                                          \
+      Operator(                                                                \
+          "aten::_set_item(Dict(" key_type ", t)(a!) l, " key_type             \
```

Comment:
I don't think this should return the dict as a result, it doesn't in python

Reply:
This was originally just mirroring the list `_set_item`, I can't see any reason why it'd be necessary to return anything though.